yb-voyager/cmd/common.go

```diff
@@ -1062,13 +1062,13 @@ type AssessmentReport struct {
 	Notes                          []string                              `json:"Notes"`
 
 	// fields going to be deprecated
-	UnsupportedDataTypes       []utils.TableColumnsDataTypes     `json:"UnsupportedDataTypes"`
-	UnsupportedDataTypesDesc   string                            `json:"UnsupportedDataTypesDesc"`
-	UnsupportedFeatures        []UnsupportedFeature              `json:"UnsupportedFeatures"`
-	UnsupportedFeaturesDesc    string                            `json:"UnsupportedFeaturesDesc"`
-	UnsupportedQueryConstructs []utils.UnsupportedQueryConstruct `json:"UnsupportedQueryConstructs"`
-	UnsupportedPlPgSqlObjects  []UnsupportedFeature              `json:"UnsupportedPlPgSqlObjects"`
-	MigrationCaveats           []UnsupportedFeature              `json:"MigrationCaveats"`
+	UnsupportedDataTypes       []utils.TableColumnsDataTypes     `json:"-"`
+	UnsupportedDataTypesDesc   string                            `json:"-"`
+	UnsupportedFeatures        []UnsupportedFeature              `json:"-"`
+	UnsupportedFeaturesDesc    string                            `json:"-"`
+	UnsupportedQueryConstructs []utils.UnsupportedQueryConstruct `json:"-"`
+	UnsupportedPlPgSqlObjects  []UnsupportedFeature              `json:"-"`
+	MigrationCaveats           []UnsupportedFeature              `json:"-"`
```

Comment:
Why not remove these fields?

Reply:
They are required by YugabyteD still since they haven't moved to the new structure yet.
cc: @sanyamsinghal

Reply:
Nvm, got it! we still need to populate these to send to YugabyteD